packages/augur-ui/test/app/actions/sync-blockchain-test.js

```diff
@@ -0,0 +1,122 @@
+import {
+  syncBlockchain,
+  __RewireAPI__ as ReWireModule
+} from "modules/app/actions/sync-blockchain";
+import { describe, it, after } from "mocha";
+
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import testState from "test/testState";
+
+describe(`modules/app/actions/sync-blockchain.js`, () => {
+  // eslint-disable-line func-names, prefer-arrow-callback
+  const middlewares = [thunk];
+  const mockStore = configureMockStore(middlewares);
+  const state = Object.assign({}, testState, {
+    blockchain: {
+      currentBlockTimestamp: 4886718335,
+      currentBlockNumber: 9999,
+      currentAugurTimestamp: 42
+    },
+    gasPriceInfo: {
+      blockNumber: undefined
+    }
+  });
+  const dataReturned = {
+    currentBlockNumber: 0x10000,
+    currentBlockTimestamp: 0x4886718345,
+    currentAugurTimestamp: 42,
+  };
+  const syncData = {
+    highestBlock: 22,
+    lastProcessedBlock: 22,
+  }
+  const store = mockStore(state);
+  const AugurJS = {
+    rpc: {
+      getCurrentBlock: () => ({
+        number: 10000,
+        timestamp: 4886718345
+      }),
+      block: {
+        number: 10000,
+        timestamp: 4886718345
+      }
+    },
+    api: {
+      Controller: {
+        getTimestamp: callback => {
+          callback(null, 42);
+        }
+      }
+    },
+    augurNode: {
+      getSyncData: callback => {
+        callback(null, {
+          highestBlock: {
+            number: 22
+          },
+          lastProcessedBlock: {
+            number: 22
+          },
+        })
+      }
+    },
+  };
+
+  const updateBlockchain = data => ({
+    type: "UPDATE_BLOCKCHAIN",
+    data
+  });
+  const updateAssets = data => ({
+    type: "UPDATE_ASSETS"
+  });
+  const loadGasPriceInfo = () => ({
+    type: "UPDATE_GAS_PRICE_INFO"
+  });
+  ReWireModule.__Rewire__("augur", AugurJS);
```

Comment:
@bthaile These will need to be rewritten.

Reply:
This test will get migrated on augur-ui master then we'll merge it over.